### PR TITLE
Iui-33

### PR DESCRIPTION
**Pull request by mstone6769**
IUI-33 - Adding the ability to customize the labels (foreign language support)

### CODE REVIEW

File: lib/src/directives/iui-pager.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ angular.module('iui.pager', [])
     },
     templateUrl : '/$iui-table/templates/iui-pager.html',
     link : function(scope, element, attrs){
+      scope.labels=iuiTableLabels;
```
Comment (mcastre) [inline lib/src/directives/iui-pager.js]: spacing here

File: lib/src/directives/iui-table.js [hunk 2]
```diff
@@ -205,59 +205,46 @@ app.controller('TableController2', [function () {
...
+        'rowData',
+        'tableCaption',
+        'hideTableCaption',
+        'hideTablePager',
+        'rowTemplate',
+        'tableClass',
+        'serverSideSorting'
+      ];
+
+      _.each(watchAttributes, function(attributeName) {
```
Comment (mcastre) [inline lib/src/directives/iui-table.js]: nicely done

File: lib/src/directives/iui-table.test.js [hunk 3]
```diff
@@ -274,6 +276,26 @@
...
 
+      describe('language support', function() {
+        it('should have English defaults', function() {
+          pagination.firstButton = el.find('button').eq(0);
+          expect(pagination.firstButton.text().trim()).toBe('First');
+
+        });
+        it('should allow you to customize the sr-only text for buttons through the iuiTableLabels service', function() {
+          iuiTableLabels.first = 'Primero';
+          iuiTableLabels.last = 'Último';
```
Comment (mcastre) [inline lib/src/directives/iui-table.test.js]: haha nice

File: lib/src/directives/iui-pager.js (hunk 1 above)
Comment (mstone6769) [inline lib/src/directives/iui-pager.js]: are you asking to add spaces before and after the equals?

File: lib/src/directives/iui-table.test.js [hunk 4]
```diff
@@ -274,6 +276,26 @@
...
+          iuiTableLabels.sortBy = 'Ordenar Por';
+          scope.$digest();
+          pagination.firstButton = el.find('button').eq(0);
+          pagination.lastButton = el.find('button').eq(3);
+          iuiTable.thElements = el.find('th');
+          expect(pagination.firstButton.text().trim()).toBe('Primero');
+          expect(pagination.lastButton.text().trim()).toBe('Último');
+          expect(iuiTable.thElements.eq(0).text().trim().replace(/(\r\n|\n|\r)/gm,'').replace(/\s+/g,' ')).toBe('Ordenar Por Code Name');
```
Comment (mcastre) [inline lib/src/directives/iui-table.test.js]: if you are using an English term in a Spanish sentence, it needs to be in quotes. Future reference.

**Comment by mcastre** [comment]
We're going to have to add documentation on how to customize the labels in a different language.

File: lib/src/directives/iui-pager.js (hunk 1 above)
Comment (mcastre) [inline lib/src/directives/iui-pager.js]: yea. Keep it consistent.

**Pull request merged**